### PR TITLE
Mob 1705 default browser experiment

### PR DESCRIPTION
**Pull request by ecotopian**

### CODE REVIEW

File: Client/Ecosia/Settings/EcosiaDebugSettings.swift [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+    }
+
+    override func onClick(_ navigationController: UINavigationController?) {
+        User.shared.treeCount += 10
+        self.settings.tableView.reloadData()
+        NotificationCenter.default.post(name: .HomePanelPrefsChanged, object: nil)
+    }
+}
+
+final class UnleashDefaultBrowserSetting: HiddenSetting {
```
Comment (ecotopian) [inline Client/Ecosia/Settings/EcosiaDebugSettings.swift:178]: This setting is new

File: Client/Ecosia/Settings/EcosiaDebugSettings.swift [hunk 2]
```diff
@@ -0,0 +1,209 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import UIKit
+import Core
+import Shared
+
+final class PushBackInstallation: HiddenSetting {
```
Comment (ecotopian) [inline Client/Ecosia/Settings/EcosiaDebugSettings.swift:10]: Those Settings were moved here. So no need to review them to deeply.

File: Client/Ecosia/Settings/EcosiaDebugSettings.swift [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+                _ = try await Unleash.reset(env: .staging)
+            } catch {
+                debugPrint(error)
+            }
+            await MainActor.run {
+                self.settings.tableView.reloadData()
+
+                let alert = AlertController(title: "New variant assigned", message: "It might be the old one though", preferredStyle: .alert)
+                navigationController?.topViewController?.present(alert, animated: true) {
+                    DispatchQueue.main.asyncAfter(deadline: .now() + 1.5) {
```
Comment (jman82) [inline Client/Ecosia/Settings/EcosiaDebugSettings.swift:202]: Is "1.5" the time in seconds? Just curious

**Review by jman82 — APPROVED** [review]
Looks great :rocket:

File: Client/Frontend/Browser/BrowserViewController.swift [hunk 4]
```diff
@@ -2243,6 +2242,26 @@ extension BrowserViewController {
             present(intro, animated: true)
             User.shared.hideRebrandIntro()
         }
```
Comment (rashmi-ecosia) [inline Client/Frontend/Browser/BrowserViewController.swift:2244]: So rebrand intro pop up is still shown at the start for new users irrespective of when the default browser pop up is timed?

File: Client/Ecosia/Settings/EcosiaDebugSettings.swift (hunk 3 above)
Comment (ecotopian) [reply Client/Ecosia/Settings/EcosiaDebugSettings.swift:202]: Yes, seconds 👍

File: Client/Frontend/Browser/BrowserViewController.swift (hunk 4 above)
Comment (ecotopian) [reply Client/Frontend/Browser/BrowserViewController.swift:2244]: Great question! In line 2255 in the same file, you'll see that there is an additional check inside `presentDefaultBrowserPromoIfNeeded()` for the Rebrand intro. So in that case it won't be shown.

**Review by rashmi-ecosia — APPROVED** [review]
🚀

**Pull request merged**